attestation-agent/attester/src/tdx/mod.rs

```diff
@@ -9,7 +9,6 @@ use crate::utils::pad;
 use crate::InitdataResult;
 use anyhow::*;
 use base64::Engine;
-use log::debug;
```

Comment:
This might be a leftover of some commit

Reply:
actually, it's on purpose to handle macros scope consistently because with  `use log:: {debug, warn}` and https://github.com/confidential-containers/guest-components/pull/572/commits/1c8291f20d801a1242c0cf7c549fa6a04a29b333 combined it triggers unused imports.